Add lmmendes/game-boy-opcodes to Cheatsheets, Tables section

lmmendes opened the pull request:
Adding lmmendes/game-boy-opcodes to Cheatsheets, Tables section.

It's basically a JSON representation of the http://www.pastraiser.com/cpu/gameboy/gameboy_opcodes.html for ease of parsing.

The file also includes some opcode length and timing fixes for known issues with the pastraiser.com original content.

avivace commented:
1. If you have fixes, propose them on an equivalent format of the same document
2. Why it is relevant to have a JSON representation of something already existent on the list as a separate entry?

Supposing this is *only* a JSON format of something already there (which is not, at the moment), I'd suggest to have that as

- [Existent link]() - description ([JSON]() version)

lmmendes commented:
Updated the README.md as suggested. Removed the new entry and added the link for the JSON version of the page has a alternative format.

The value of the having a JSON version of the page is that it makes easy to build a parser to scaffold  template pages with the needed functions corresponding to the opcodes to implement.

Example of the output of processing the JSON file to generate a ruby stub for a gameboy emulator that I'm currently writing: 

```
      # INC BC
      # flags: - - - -
      def inc_bc
      end

      # INC B
      # flags: Z 0 H -
      def inc_b
      end

      # DEC B
      # flags: Z 1 H -
      def dec_b
      end
```
